libs/langchain/langchain/schema/memory.py

```diff
@@ -48,7 +49,9 @@ def memory_variables(self) -> List[str]:
         """The string keys this memory class will add to chain inputs."""
 
     @abstractmethod
-    def load_memory_variables(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
+    def load_memory_variables(
+        self, inputs: Dict[str, Any], callbacks: Callbacks = None
```

Comment:
@nfcampos should I update the `callbacks` typing to accept a RunnableConfig and diff off yours?